dkpro-core-opennlp-asl/src/scripts/build.xml

```diff
@@ -295,6 +304,8 @@
                 <entry key="DC.title" value="de-chunker-opennlp.bin"/>
                 <entry key="DC.creator" value="LTL, University of Duisburg-Essen"/>
                 <entry key="DC.identifier" value="http://www.ltl.uni-due.de/content/6-software/de-chunker-opennlp.bin"/>

```

Comment:
@reckart Looks better if DC identifier should be similar to  src url, so ```http``` should be updated with ```https```.

Reply:
Please make the update ;)